### ISSUE TITLE
windows: Get-WmiObject doesnt work in powershell 7? 

### ISSUE THREAD

**Issue by jayunit100** [projectcalico/calico#4255]
## Expected Behavior 

Powershell installer would work on powershell v7

## Current Behavior
Sems like the ` Get-WmiObject win32` is calling an old powershell library . ? Not really sure, but it doesnt seem to work 

```
Set parameter "AppRotateBytes" for service "CalicoFelix".
Done installing Felix service.
Copying CNI binaries into place.
Writing CNI configuration to c:\k\cni\config\10-calico.conf.
Wrote CNI configuration.
Starting Calico...
This may take several seconds if the vSwitch needs to be created.
Waiting for Calico initialisation to finish...
Get-WmiObject: C:\CalicoWindows\libs\calico\calico.psm1:299
Line |
 299 |      $bootTime = (Get-WmiObject win32_operatingsystem | select @{LABEL …
     |                   ~~~~~~~~~~~~~
     | The term 'Get-WmiObject' is not recognized as a name of a cmdlet, function, script file, or executable
     | program. Check the spelling of the name, or if a path was included, verify that the path is correct
     | and try again.

Exception: C:\CalicoWindows\libs\calico\calico.psm1:302
Line |
 302 |          throw "Failed to get last boot time"
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Failed to get last boot time
```

## Possible Solution 

Maybe run an old powershell version when installing calico.psm1 

## Steps to Reproduce (for bugs)
1. try to run https://docs.projectcalico.org/getting-started/windows-calico/kubernetes/standard#install-calico-and-kubernetes-on-windows-nodes on a new version of windows server w/ powershell 7

## Context

PLayin  around w/ calico for k8s on windows 

## Your Environment

* Calico version 1.17
* Orchestrator version (e.g. kubernetes, mesos, rkt): k8s 1.19
* Operating System and version: windows 2019

**Comment by perithompson** [comment]
@jayunit100 Powershell 7 doesn't contain WMI interfaces as it is a cross-platform version.  At the current time, you will need to use the Powershell version that is contained in the WMF 5.1

**Comment by jayunit100** [comment]
yeah we can make a PR to fix gthis i guess, just break if users try ps 7

**Comment by lmm** [comment]
@jayunit100 thanks for the report!

> A VXLan error sometimes happens, not sure why, when starting up felix

Hmm... did the error go away and was networking working?

> appears that we're using the CNI plugin from the older

Yeah we need to add support for containerd

> more info on debugging kube-proxy in cases where it doesnt work (i can make an upstream PR to help w/ this)

What issues did you encounter with kube-proxy?

**Comment by lmm** [comment]
> Powershell installer would work on powershell v7 

Thanks, we'll need to fix this
